internal/controllers/rest_controller.go

```diff
@@ -0,0 +1,122 @@
+package controllers
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/coreos/go-oidc"
+	"github.com/gin-gonic/gin"
+	"golang.org/x/oauth2"
+
+	"github.com/ydataai/authentication-service/internal/clients"
+	"github.com/ydataai/authentication-service/internal/services"
+	"github.com/ydataai/go-core/pkg/common/logging"
+	"github.com/ydataai/go-core/pkg/common/server"
+)
+
+// RESTController defines rest controller
+type RESTController struct {
+	restService   *services.RESTService
+	configuration RESTControllerConfiguration
+	oidcClient    *clients.OIDCClient
+	logger        logging.Logger
+}
+
+// NewRESTController initializes rest controller
+func NewRESTController(restService *services.RESTService,
+	configuration RESTControllerConfiguration,
+	oidcClient *clients.OIDCClient,
+	logger logging.Logger) RESTController {
+	return RESTController{
+		restService:   restService,
+		configuration: configuration,
+		oidcClient:    oidcClient,
+		logger:        logger,
+	}
+}
+
+// Boot ...
+func (rc RESTController) Boot(s *server.Server) {
+	s.Router.GET("/healthz", rc.healthCheck())
+
+	s.Router.GET(rc.configuration.AuthServiceURL, gin.WrapF(rc.RedirectAuthEndpoint))
+	s.Router.GET(rc.configuration.OIDCCallbackURL, gin.WrapF(rc.Callback))
+	// s.Router.POST(rc.configuration.LogoutURL, gin.WrapF(rc.Logout))
```

Comment:
Remove commented line.

Reply:
i saw that this is already in the configuration, so you can use it already

Reply:
I've been thinking about it and I think it's better not to deal with the `/logout` endpoint in this PR because this needs to be discussed. I will remove it.